stdpopsim/cli.py

```diff
@@ -708,20 +721,40 @@ def run_simulation(args):
                     "A DFE annotation has been assigned without a DFE. "
                     "Please specify a DFE."
                 )
+            elif args.dfe_bed_file is not None:
```

Comment:
This and the one above should probably be `if`, not `elif`, for clarity.

Reply:
done